packages/Avatar/stories/examples/SizesColors.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Story } from "storybook/assets/styles/common.styles";
+import Heading from "@paprika/heading";
+import Avatar from "../../src";
+import getAvatarColors from "../../src/helpers";
+
+const text = [
+  "apple",
+  "box",
+  "cake",
+  "duck",
+  "earth",
+  "flower",
+  "glasses",
+  "hair",
+  "image",
+  "jellybeans",
+  "kayak",
+  "lamp",
+];
+
+export default () => {
+  return (
+    <Story>
+      <Heading level={3}>Colors</Heading>
+      <div style={{ display: "flex", flexWrap: "wrap" }}>
```

Comment:
for use only on stories, instead of style prop you can make use of the css prop provided by styled-components.

```js
<div css={`display: flex; flexWrap: wrap;`}>
```

Reply:
would this be syntax highlighted btw?